public-ui/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "aou-public-ui-angular",
+  "version": "0.0.2",
+  "license": "BSD",
+  "scripts": {
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "test": "ng test --sourcemaps=false",
+    "lint": "ng lint --type-check",
+    "e2e": "ng e2e",
+    "codegen": "yarn run codegen-clean && yarn run public-codegen-swagger && yarn run public-post-codegen",
+    "merge-swagger": "../api/build.rb merge-yaml",
+    "codegen-clean": "rm -rf src/generated src/publicGenerated",
+    "post-codegen": "sed -i.bak -e 's/OpaqueToken/InjectionToken/' src/generated/variables.ts",
+    "public-codegen-swagger": "java -jar ../aou-utils/swagger-codegen-cli.jar generate --lang typescript-angular --input-spec ../public-api/src/main/resources/public-api.yaml --output src/publicGenerated --additional-properties ngVersion=2",
+    "public-post-codegen": "sed -i.bak -e 's/OpaqueToken/InjectionToken/' src/publicGenerated/variables.ts",
+    "coverage": "ng test --sourcemaps=false --watch=false --code-coverage"
+  },
+  "private": true,
+  "dependencies": {
+    "@angular-redux/store": "^6.6.0",
```

Comment:
prune? Seems unlikely that you need most of these: e.g. redux, clarity, d3, highcharts, immutable, ng2-nouislider

Reply:
Pruned. I use highcharts fyi. Got the ok on it.